PixaPicker/ViewController.swift

```diff
@@ -7,16 +7,33 @@
 //
 
 import UIKit
-
 import SDWebImage
 
-class ViewController: UIViewController {
+class ViewController: UIViewController, UISearchResultsUpdating, UISearchBarDelegate {
+    
+    let apiKey = "key=13683470-874d69ddffa828cbb82551e32" //API key for my Pixabay account
     @IBOutlet weak var picsCollection: UICollectionView!
-    @IBOutlet weak var searchBar: UISearchBar!
+    
+    func updateSearchResults(for searchController: UISearchController) {
+        let searchText = searchController.searchBar.text!
```

Comment:
avoid using !  Instead, write it as `guard let searchText = searchController.searchBar.text else { return }`

Reply:
Adding onto this, its just good habits to not force unwrap (!).  You're telling the compiler you are guarenteeing there's a value there which can be dangerous, even if you feel its 100% certain, use guard statements instead.